lib/Backends/CPU/CMakeLists.txt

```diff
@@ -4,16 +4,7 @@ else()
   find_program(CLANG_BIN clang++)
 endif()
 
-if(TARGET llvm-link)
-  set(LLVM_LINK_BIN $<TARGET_FILE:llvm-link>)
-else()
-  find_program(LLVM_LINK_BIN
-               NAMES
-                 llvm-link-${LLVM_VERSION_MAJOR}
-                 llvm-link-8
-                 llvm-link-7
-                 llvm-link)
-endif()
+set(LLVM_LINK_BIN ${LLVM_BINARY_DIR}/bin/llvm-link)
```

Comment:
hmm, I *think* that this should be equivalent.

Reply:
Probably, but at least it does not try to use system's llvm-link now.